source/exploring_catalog_and_datasets/04_getting_involved/using_api.rst

```diff
@@ -30,33 +25,31 @@ These APIs can be used, for example, to search for datasets and data, to compute
    * Each record provides values for each of the dataset's fields.
 
 
-ODS Search
-----------
+ODS Explore API
+---------------
 
-This ODS Search API is used internally by the Opendatasoft platform and widgets. To develop an application, we recommend using this API.
+The ODS Explore API is used internally by the Opendatasoft platform and widgets. To develop an application, we recommend using this API, in V2.
 
-There are currently 2 versions of the ODS Search API: the `v1 <https://help.opendatasoft.com/apis/ods-search-v1/>`_ and the `v2 <https://help.opendatasoft.com/apis/ods-search-v2/>`_, which is still under development. Listed below, the main differences between the 2 versions.
+There are currently 2 versions of the ODS Explore API: the `V2 <https://help.opendatasoft.com/apis/ods-explore-v2/>`_ and the `V1 <https://help.opendatasoft.com/apis/ods-search-v1/>`_, which is now deprecated. Listed below, the main differences between the 2 versions.
 
-* The ODS Search API v2 introduces a better implementation of REST principles. Each API entry-point provides links to navigate between linked resources (HATEOAS) easily.
-* Key changes in the v2:
+* Key changes in the V2:
 
   * A richer query language (ODSQL) is available, with new functions and arithmetic expressions.
-  * Special filters (such as geofilter and facet filters) are now directly integrated to ODSQL, in the ``where`` parameter (i.e ``geofilter.distance='42,1,100'`` is replaced by ``where=distance(geo_field,geom'Point(42 1)',100)``).
-  * The aggregation API (former analyze API) now shares the same structure and ODSQL language as the Search API. It is available on both the catalog (to aggregate datasets) and the dataset (to aggregate records) APIs.
-  * Exporters have their own entry-point under ``/api/catalog/exports`` and ``/api/catalog/datasets/<dataset_id>/exports``.
-  * Navigation search (former facet search) and standard search are completely separated.
+  * Special filters (such as geofilter and facet filters) are now directly integrated to ODSQL, in the ``where`` parameter.
+  * Aggregations endpoint and records endpoint now share the same structure and ODSQL language.
```

Comment:
We agreed some time ago to deprecate the term `aggregations` which is very elastic-specific-language.
```suggestion
  * We deprecated the analyze API as all operations are now available on all endpoints. If you want to perform analysis (with expressions or functions) you can do so on either on the catalog or dataset endpoint. If you want to perform analysis on groups of records, you just have to group them with a `group_by` REST param.
```